docs/book/application-integration/usage-in-a-laminas-mvc-application.md

```diff
@@ -0,0 +1,181 @@
+# Usage in a laminas-mvc Application
+
+The following example shows _one_ potential use case of laminas-form within
+a laminas-mvc based application. The example uses a module, a controller and the
+laminas-form element manager.
+
+The example is based on the [tutorial application](https://docs.laminas.dev/tutorials/getting-started/overview/)
+which builds an album inventory system.
+
+Before starting, make sure laminas-form is installed and configured.
+
+## Create Form
+
+Create a form as separate class, e.g. `module/Album/src/Form/AlbumForm.php`:
+
+```php
+namespace Album\Form;
+
+use Laminas\Form\Element\Text;
+use Laminas\Form\Form;
+
+class AlbumForm extends Form
```

Comment:
how about implements `Zend\InputFilter\InputFilterProviderInterface`, since the form will be pulled from `FormElementManager` manager.

Reply:
The initial idea was to start with a very simple form but I added the input filter configuration for better illustration of the usage.